src/isFunction.js

```diff
@@ -0,0 +1,4 @@
+export default (value) => {
```

Comment:
why don't you use the isFunction from loadash?

https://lodash.com/docs/4.17.4#isFunction

Reply:
v5 of redux-form don't include lodash as dependency